src/Microsoft.DotNet.Interactive.Http.Tests/ParserTests.Variables.cs

```diff
@@ -299,6 +299,35 @@ public void single_quotes_in_variable_values_are_supported()
 
         }
 
+        [Fact]
+        public void double_braces_escaping_single_braces_are_supported()
+        {
+            var result = Parse(
+                """
+
+                @text=\{{text\}}
+                """
+                );
+
+            var variables = result.SyntaxTree.RootNode.TryGetDeclaredVariables().declaredVariables;
+            variables.Should().Contain(n => n.Key == "text").Which.Value.Should().BeOfType<DeclaredVariable>().Which.Value.Should().Be("{{text}}");
+        }
+
+        [Fact]
+        public void triple_escaped_curly_braces_in_variable_values_are_supported()
```

Comment:
Again, it would be useful for the test name to explain the intent and behavior more specifically. It's great when test names can read like a spec.